packages/fxa-auth-server/lib/senders/emails/templates/postAddRecoveryPhone/en.ftl

```diff
@@ -2,8 +2,8 @@ postAddRecoveryPhone-subject = Recovery phone added
 postAddRecoveryPhone-preview = Account protected by two-step authentication
 postAddRecoveryPhone-title-v2 = You added a recovery phone number
 # Variables:
-#  $maskedLastFourPhoneNumber (String) - A bullet point mask with the last four digits of the user's phone number, e.g. ••••••1234
-postAddRecoveryPhone-description-v2 = You added { $maskedLastFourPhoneNumber } as your recovery phone number
+#  $maskedPhoneNumber (String) - A bullet point mask with the last four digits of the user's phone number, e.g. ••••••1234
+postAddRecoveryPhone-description-v2 = You added { $maskedPhoneNumber } as your recovery phone number
```

Comment:
Is it necessary to change the variable name here? If so, we'll have to retrigger translations with a new ID (which seems like unnecessary work for the localizers).

Reply:
I kinda agree. I like the refactor, but it also accounts for for quite a few of the line changes. So not entirely sure it's worth it, considering it doesn't seem like the original name is misleading or anything like that.